Libs/LibUIDropDownMenu/LibUIDropDownMenu.lua

```diff
@@ -118,6 +118,10 @@ function lib:UIDropDownMenu_InitializeHelper(frame)
     frame:SetHeight(L_UIDROPDOWNMENUQUESTIE_BUTTON_HEIGHT * 2);
 end
 
+function lib:getOpen() 
+    return L_UIDROPDOWNMENUQUESTIE_OPEN_MENU
+end
```

Comment:
This is full on nitpicking: The value of `L_UIDROPDOWNMENUQUESTIE_OPEN_MENU` is pretty big table and we actually don't need it. We just need to know if it is `nil` or not. Also a small comment for the future is great, as we generally try to avoid changing dependencies and we need to make sure we are not losing that function. 

```suggestion
-- Questie addition, to know when menues are showing
function lib:getOpen() 
    return L_UIDROPDOWNMENUQUESTIE_OPEN_MENU ~= nil
end
```